doc/method/method-intro.md

```diff
@@ -24,45 +24,34 @@ Open Tree reference taxonomy version 2.11.
-    (within the same taxonomy); in
-    nomenclatural terms, either a homonym, hemihomonym, or
-    clerical error
-
-
-## *NMF suggestion on how to explain all this*
-
-1. Is it possible to assume an ideal case where merging two or more OTT input taxonomies requires no or only very minimal conflict/noise/ambiguity resolution? And the result is almost unambiguously correct? If so, perhaps you start your "assembly process" description by imaging/introducing such a case, and your core pipeline in relation to it. That is then out of the way - a scenario that OTT can handle well and easily.
-
-   [JAR: I think this is now handled by 'assembly overview' below.  Maybe the examples are too simple? Let me know.]
-
-2. Complications, 1 - those complications that through various profound or pragmatic solution you can address to a fairly large degree of satisfaction. Outcome -- still rather sound OTT, but drawing now on a full scope of things you've added because you've had to given case 1. was not what the input looked like.
-
-   [JAR: I think this is what the alignment and merge sections are about.]
-
-3. Complications, 2 - issues that you either handle not to your own satisfaction, or simply cannot handle at all.
-
-   [JAR: yes, but this belongs in the discussion section.]
-
-I guess I am suggesting this because 1 & 2 give you an opportunity to shine first, and somewhat conclusively, for a significant subset of the input trees. At least for the purpose of mounting the narrative. Clearly any complete OTT assembly job will encounter everything. But you may not have to write such that you directly follow what I assume may be real -- every input taxonomy has instances 1, 2, 3 represented to varying degrees, or they arise as the OTT grows. Instead you could pretend that some input taxonomies are clean (1), individually and jointly. Or clean enough (2) - because of your work. And only 3 is the tough stuff - but tough for anybody, etc.
-
-So, I wonder what would happen if you did this kind of thing. "For the sake of making this assembly process accessible to a wide readership, we first illustrate the entire pipeline when acting on two or more input taxonomies that are highly internally consistent, and also pose minimal conflict among them. Here the assembly works well from A to Z, as we show and exemplify.
-
-"A second category are complications that occur frequently but for which we have developed adequate diagnosis and repair/resolution mechanisms. We show how we do this, and also show what else could be done for even better performance".
-
-"A third category contains lingering challenges that point to future solution analysis/development needs. And we suggest ..."
-
-(and of course you'd say that in reality, every input may be a mix of 1-3)
+  * homonym = where a single name-string belongs to multiple nodes
+    (within the same taxonomy).  This is the nontechnical meaning of 'homonym'
```

Comment:
no. it wouldn't make any sense; you can only talk about the number of nodes with a given name when talking about a single taxonomy.  otherwise, anything that occurred in two taxonomies would be a homonym.